packages/logging/charts/logging/templates/flow.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: logging.banzaicloud.io/v1beta1
```

Comment:
Are clusterOutput and clusterFlow just here for the default setup ?

Reply:
Yes they are only used for the default cluster flow.